Source/Orts.Simulation/Simulation/Physics/Train.cs

```diff
@@ -247,6 +247,7 @@ public enum TRAINTYPE
         public float maxTimeS = 120;                     // check ahead for distance covered in 2 mins.
         public float minCheckDistanceM = 5000;           // minimum distance to check ahead
         public float minCheckDistanceManualM = 3000;     // minimum distance to check ahead in manual mode
+        public float minCheckDistanceExplorerM => Math.Max(AllowedMaxSpeedMpS * 240, 5000);      // minimum distance to check ahead in explorer mode
```

Comment:
```suggestion
        /// <summary>
        /// Minimum distance to check ahead in explorer mode.
        /// </summary>
        private float minCheckDistanceExplorerM
        {
            get
            {
                const float checkTimeS = 240, checkDistanceM = 5000;
                return Math.Max(AllowedMaxSpeedMpS * checkTimeS, checkDistanceM);
            }
        }
```
A few things I'd like to see changed here:
- We want to avoid public instance variables if we can help it.
- Annotate the `240` and `5000` constants.
- Move the variable description to a proper XML comment. This is required to get help text working in Intellisense; it's not just a stylistic choice. (Yes, almost all of our current code violates this standard, but you've got to start somewhere.)

Reply:
Do I change the case of the property name?

Reply:
The C# guideline (enforced by Visual Studio) is that instance variable names need to use PascalCase, but *technically* that only applies to public, not private, variable names.

Still, I think it would be a good idea to change the case. Do use the auto-rename tool - I can't make that change in a GitHub suggestion.